.storybook/main.ts

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  addons: [
+    "@storybook/addon-backgrounds/register",
+    "@storybook/addon-knobs/register",
+    "@storybook/addon-notes/register-panel",
+    "@storybook/addon-a11y/register"
+  ],
+  presets: ["@storybook/addon-docs/preset"],
+  stories: ["../src/**/*.stories.(mdx|ts|js)"]
```

Comment:
oh nice, mdx support will be great

Reply:
Yes, we're using these to have some documentation generated from the markdown files.